lib/horde/dynamic_supervisor_impl.ex

```diff
@@ -376,20 +364,61 @@ defmodule Horde.DynamicSupervisorImpl do
 
   def has_membership_change?([]), do: false
 
+  defp handoff_processes(state) do
+    this_node = fully_qualified_name(state.name)
+
+    Map.values(state.processes_by_id)
+    |> Enum.reduce(state, fn {current_node, child, _child_pid}, state ->
+      case choose_node(child, state) do
+        {:ok, %{name: chosen_node}} ->
+          current_member = Map.get(state.members_info, current_node)
+
+          cond do
+            this_node != current_node and this_node == chosen_node ->
+              # handle_dead_nodes 
+              case current_member do
+                %{status: :dead} ->
+                  {{:ok, _}, state} = add_child(child, state)
+                  state
+
+                _ ->
+                  state
+              end
+
+            this_node == current_node and chosen_node != this_node ->
+              case Application.get_env(:horde, :handoff_on_alive, true) do
```

Comment:
Will do this today!

Reply:
Resolved in [692028a](https://github.com/derekkraan/horde/pull/164/commits/692028ab2aed36e5f36148b7ca29be21ff6de286) 

Since I ended up going with an options structure that is non-boolean (you either set :process_redistribution to :active or :passive) I've kept the case statement because it's more expressive.